yb-voyager/cmd/exportSchema.go

```diff
@@ -226,27 +225,32 @@ func applyMigrationAssessmentRecommendations() error {
 	}
 
 	log.Infof("parsing assessment report json file for applying recommendations")
-	var report AssessmentReport
-	err := jsonfile.NewJsonFile[AssessmentReport](assessmentReportPath).Load(&report)
+	report, err := ParseJSONToAssessmentReport(assessmentReportPath)
 	if err != nil {
 		return fmt.Errorf("failed to parse json report file %q: %w", assessmentReportPath, err)
 	}
 
-	err = applyColocatedVsShardedTableRecommendation(report.Sizing)
+	shardedTables, err := report.GetShardedTablesRecommendation()
 	if err != nil {
-		return fmt.Errorf("failed to apply colocated vs sharded table recommendation: %w", err)
+		log.Warnf("GetShardedTablesRecommendation: %v", err)
```

Comment:
if you look the `report.GetShardedTablesRecommendation()`
It returns error in case if shardedTables list is null, no other error case.

Is your point is why are we returning that as an error instead of just handing the null value here?

Reply:
Oh okay. In that case, yes,  I'd suggest checking for nil value, instead of handling it as an error. (it's not really an error).

Reply:
make sense. will do the change.